src/Microsoft.AspNet.Mvc.ModelBinding/Validation/CompositeClientModelValidatorProvider.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
+{
+    /// <summary>
+    /// Default implementation for <see cref="ICompositeClientModelValidatorProvider"/>.
+    /// </summary>
+    public class CompositeClientModelValidatorProvider : ICompositeClientModelValidatorProvider
+    {
+        /// <summary>
+        /// Initializes a new instance of <see cref="CompositeClientModelValidatorProvider"/>.
+        /// </summary>
+        /// <param name="providers">
+        /// A collection of <see cref="IClientModelValidatorProvider"/> instances.
+        /// </param>
+        public CompositeClientModelValidatorProvider([NotNull] IEnumerable<IClientModelValidatorProvider> providers)
+        {
+            ValidatorProviders = new List<IClientModelValidatorProvider>(providers);
+        }
+
+        public IReadOnlyList<IClientModelValidatorProvider> ValidatorProviders { get; }
+
+        /// <inheritdoc />
+        public void GetValidators(ClientValidatorProviderContext context)
+        {
+            foreach (var validatorProvider in ValidatorProviders)
+            {
+                validatorProvider.GetValidators(context);
+            }
+        }
```

Comment:
I don't feel like this class is really needed since we're removing these kinds of things already. Let's discuss if you think it's important to have.